src/stories/containers/TransparencyReport/utils/descriptionsConstants.ts

```diff
@@ -0,0 +1,2 @@
+export const CANONNICAL_DESCRIPTION =
+  "Canonical Expense Categories in MakerDAO are standardized classifications of expenses tailored for managing MakerDAO's operational costs.";
```

Comment:
This will probably be better to add it to the site it is needed as it is not reused

Reply:
ok, i will